src/i18n/LanguageManager.js

```diff
@@ -0,0 +1,99 @@
+/**
```

Comment:
just a small change here, the name of the file should be `languageManager.js` understanding that manage only javascript logic and not a react component.